test/src/androidTest/java/co/condorlabs/customcomponents/test/EditTextPasswordFieldTest.kt

```diff
@@ -75,34 +76,59 @@ class EditTextPasswordFieldTest : MockActivityTest() {
         not(isTextDisplayed("1234567890"))
     }
 
+    @SmallTest
     @Test
     fun eyeIconIsDisplay() {
-        Espresso.onView(allOf(withId(R.id.text_input_password_toggle), isDisplayed()))
+        restartActivity()
+        // Given
+        val view = (ruleActivity.activity.findViewById<View>(R.id.tlPassword) as? EditTextPasswordField)
+        val textInpuntLayout = view!!.textInputLayout!!
```

Comment:
Use safe call (?) instead. Check in the Assert if the `textInputLayout` is null

Reply:
done